connectors/src/connectors/google_drive/lib/hierarchy.ts

```diff
@@ -13,12 +13,13 @@ async function getFileParents(
   driveFile: GoogleDriveObjectType,
   /* eslint-disable @typescript-eslint/no-unused-vars */
   startSyncTs: number
-): Promise<GoogleDriveObjectType[]> {
+): Promise<string[]> {
```

Comment:
@tdraier nit: could you add a comment to that fn, explaining that it returns parents in order, with parents[0] === file, to follow convention? (can be done in follow up PR if needed / or I can take care of it)

since we always wonder if it's done correctly, let's state it loud and clear here so it's good for microsoft